pokedex_async_redux/lib/state/action/pokemon_actions.dart

```diff
@@ -13,3 +13,14 @@ class GetPokemonsAction extends ReduxAction<AppState> {
 
     return state.copyWith(pokemons: pokemons);
   }
+}
+
+/// Getting of a specific Pokemon and its details
+class GetPokemonDetailsAction extends ReduxAction<AppState> {
+  @override
+  Future<AppState> reduce() async {
+    final pokemonDetails = await ApiService().pokemonApi.getPokemonDetails(name: 'snorlax');
```

Comment:
name should be dynamic instead of fix value, kindly utilize constructor param injection for `pokemonName` 🙂

Reply:
put the snorlax value in a constant and have a todo comment as well if ever 🙂